build-logic/src/main/kotlin/elmslie.base-lib.gradle.kts

```diff
@@ -0,0 +1,12 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
```

Comment:
Do we want this convention be separate?

Reply:
yeah, as it is shared between kotlin lib and android lib conventions